src/parts/Topbar/index.tsx

```diff
@@ -15,7 +15,7 @@ import {
   tickerToCurrencyIdLiteral
 } from '@interlay/interbtc-api';
 
-import Balances from './Balances';
+import { Tokens } from 'components/Tokens';
```

Comment:
@tomjeatt 
I guess it could be `import Tokens from 'components/Tokens';` since we've been following this convention.